schemas/20-cloud-extra.sql

```diff
@@ -3,47 +3,21 @@
 
+-- More partitions can be added by "reorganizing" the pDefault parition. E.g.:
+/*
+ALTER TABLE JobRecords REORGANIZE PARTITION pDefault INTO (
+    PARTITION p2016_01 VALUES LESS THAN (TO_DAYS('2018-01-01')),
+    PARTITION p2016_02 VALUES LESS THAN (TO_DAYS('2018-02-01')),
+    PARTITION p2016_03 VALUES LESS THAN (TO_DAYS('2018-03-01')),
+    PARTITION pDefault VALUES LESS THAN MAXVALUE
+);
+*/
+
 ALTER TABLE CloudRecords
 PARTITION BY RANGE (TO_DAYS(StartTime)) (
-PARTITION p2012_12 VALUES LESS THAN (TO_DAYS('2013-01-01')),
-PARTITION p2013_01 VALUES LESS THAN (TO_DAYS('2013-02-01')),
-PARTITION p2013_02 VALUES LESS THAN (TO_DAYS('2013-03-01')),
-PARTITION p2013_03 VALUES LESS THAN (TO_DAYS('2013-04-01')),
-PARTITION p2013_04 VALUES LESS THAN (TO_DAYS('2013-05-01')),
-PARTITION p2013_05 VALUES LESS THAN (TO_DAYS('2013-06-01')),
-PARTITION p2013_06 VALUES LESS THAN (TO_DAYS('2013-07-01')),
-PARTITION p2013_07 VALUES LESS THAN (TO_DAYS('2013-08-01')),
-PARTITION p2013_08 VALUES LESS THAN (TO_DAYS('2013-09-01')),
-PARTITION p2013_09 VALUES LESS THAN (TO_DAYS('2013-10-01')),
-PARTITION p2013_10 VALUES LESS THAN (TO_DAYS('2013-11-01')),
-PARTITION p2013_11 VALUES LESS THAN (TO_DAYS('2013-12-01')),
-PARTITION p2013_12 VALUES LESS THAN (TO_DAYS('2014-01-01')),
-PARTITION p2014_01 VALUES LESS THAN (TO_DAYS('2014-02-01')),
-PARTITION p2014_02 VALUES LESS THAN (TO_DAYS('2014-03-01')),
-PARTITION p2014_03 VALUES LESS THAN (TO_DAYS('2014-04-01')),
-PARTITION p2014_04 VALUES LESS THAN (TO_DAYS('2014-05-01')),
-PARTITION p2014_05 VALUES LESS THAN (TO_DAYS('2014-06-01')),
-PARTITION p2014_06 VALUES LESS THAN (TO_DAYS('2014-07-01')),
-PARTITION p2014_07 VALUES LESS THAN (TO_DAYS('2014-08-01')),
-PARTITION p2014_08 VALUES LESS THAN (TO_DAYS('2014-09-01')),
-PARTITION p2014_09 VALUES LESS THAN (TO_DAYS('2014-10-01')),
-PARTITION p2014_10 VALUES LESS THAN (TO_DAYS('2014-11-01')),
-PARTITION p2014_11 VALUES LESS THAN (TO_DAYS('2014-12-01')),
-PARTITION p2014_12 VALUES LESS THAN (TO_DAYS('2015-01-01')),
-PARTITION p2015_01 VALUES LESS THAN (TO_DAYS('2015-02-01')),
-PARTITION p2015_02 VALUES LESS THAN (TO_DAYS('2015-03-01')),
-PARTITION p2015_03 VALUES LESS THAN (TO_DAYS('2015-04-01')),
-PARTITION p2015_04 VALUES LESS THAN (TO_DAYS('2015-05-01')),
-PARTITION p2015_05 VALUES LESS THAN (TO_DAYS('2015-06-01')),
-PARTITION p2015_06 VALUES LESS THAN (TO_DAYS('2015-07-01')),
-PARTITION p2015_07 VALUES LESS THAN (TO_DAYS('2015-08-01')),
-PARTITION p2015_08 VALUES LESS THAN (TO_DAYS('2015-09-01')),
-PARTITION p2015_09 VALUES LESS THAN (TO_DAYS('2015-10-01')),
-PARTITION p2015_10 VALUES LESS THAN (TO_DAYS('2015-11-01')),
-PARTITION p2015_11 VALUES LESS THAN (TO_DAYS('2015-12-01')),
-PARTITION p2015_12 VALUES LESS THAN (TO_DAYS('2016-01-01')),
-PARTITION p2016_01 VALUES LESS THAN (TO_DAYS('2016-02-01')),
-PARTITION p2016_02 VALUES LESS THAN (TO_DAYS('2016-03-01')),
+PARTITION pNull VALUES LESS THAN (0),
+-- Partition pNull will contain Records with a NULL
```

Comment:
I can see how you might want to split this partition from the rest, but comments usually come before the line they refer to.

Reply:
changed